vm/src/org/mozartoz/truffle/nodes/local/InitializeTmpNode.java

```diff
@@ -8,11 +8,11 @@
 public class InitializeTmpNode extends OzNode implements FrameSlotNode {
 
 	@Child WriteFrameSlotNode writeFrameSlotNode;
-	@Child OzNode value;
+	@Child OzNode valueNode;
 
-	public InitializeTmpNode(FrameSlot slot, OzNode value) {
+	public InitializeTmpNode(FrameSlot slot, OzNode valueNode) {
 		this.writeFrameSlotNode = WriteFrameSlotNodeGen.create(slot);
-		this.value = value;
+		this.valueNode = valueNode;
```

Comment:
What about the tryDeref? That was a good idea I think. Or is it never useful here?